content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -85,7 +85,6 @@ ptfe$ cat settings.json
         "value": "poc"
     },
     "placement": {},
-    "postgres_url": {},
```

Comment:
Should this output also be updated with the replacements?

Reply:
I thought about that, but this output is really just a representative sample (and not a very good one, at that).  I'll revisit this example output.